### DISCUSSION TITLE
3 hour interval in CupertinoTimerPicker

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#2971] [Q&A]
### Question

The problem is that if you use the widget 
CupertinoTimerPicker or CupertinoDatePicker
And try to flip through the clock, they flip through 3 hours at a time.

upd: I've noticed that if you turn the wheel sharply.
Time scale as if it starts to "Stutter" and scrolls for 1-2 hours. 

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Will you mind providing a runnable code sample?

**Reply by RRozi** [reply]
You can use any of the code from the official examples

**Reply by ndonkoHenri** [reply]
The thing is, I don't even understand what you mean...

**Reply by RRozi** [reply]
And what's not clear? When you scroll through the "hour" field, it will scroll 3 hours at once, not one at a time

**Comment by RRozi** [comment]
АУУУУУУУУУУУУУУУУУ

**Comment by RRozi** [comment]
Help Pleas

**Reply by baseplate-admin** [reply]
please send a minimum reproduction?

**Reply by RRozi** [reply]
Official example. And if you move the mouse wheel a little, it immediately scrolls through 3 hours, the same with the minutes 

```python
import time
import flet as ft

def main(page):
    page.theme_mode = ft.ThemeMode.LIGHT
    page.horizontal_alignment = ft.CrossAxisAlignment.CENTER

    timer_picker_value_ref = ft.Ref[ft.Text]()

    def handle_timer_picker_change(e):
        val = int(e.data)
        timer_picker_value_ref.current.value = time.strftime(
            "%H:%M:%S", time.gmtime(val)
        )
        page.update()

    timer_picker = ft.CupertinoTimerPicker(
        value=3600,
        second_interval=10,
        minute_interval=1,
        mode=ft.CupertinoTimerPickerMode.HOUR_MINUTE_SECONDS,
        on_change=handle_timer_picker_change,
    )

    page.add(
        ft.Row(
            tight=True,
            controls=[
                ft.Text("TimerPicker Value:", size=23),
                ft.TextButton(
                    content=ft.Text("00:01:10", size=23, ref=timer_picker_value_ref),
                    style=ft.ButtonStyle(color=ft.colors.RED),
                    on_click=lambda _: page.show_bottom_sheet(
                        ft.CupertinoBottomSheet(
                            timer_picker,
                            height=216,
                            bgcolor=ft.cupertino_colors.SYSTEM_BACKGROUND,
                            padding=ft.padding.only(top=6),
                        )
                    ),
                ),
            ],
        ),
    )

ft.app(target=main)
```

**Reply by RRozi** [reply]
Help..../

**Reply by RRozi** [reply]
Any Cupertino field will scroll 3 at a time

**Comment by RRozi** [comment]
Any Cupertino field will scroll 3 at a time

**Comment by ndonkoHenri** [comment]
Here is what I get:
![picker-ezgif com-video-to-gif-converter](https://github.com/flet-dev/flet/assets/98978078/0e7684bd-f48b-43cf-a9ba-48e94e7fe385)

**Reply by RRozi** [reply]
Do you have a mac? I have a web application where they can log in from any device

**Reply by ndonkoHenri** [reply]
Yep, I have one.

**Reply by RRozi** [reply]
So what should I do? It's strange why I'm scrolling through 3 cells at once

**Reply by ndonkoHenri** [reply]
Can you share a screen record of the issue? And perhaps some repro code too?

**Comment by Al-landmine** [comment]
"I am also having the same problem. The cause is the mouse scroll amount setting in Windows, but I would like to know how to fix it without changing the Windows settings."

**Reply by ndonkoHenri** [reply]
I don't know if Flutter makes it possible to control that. You can search and share feedback here.

**Comment by ndonkoHenri** [comment]
Lets further discuss this here: https://github.com/flet-dev/flet/issues/3649
